server-readonly/pom.xml

```diff
@@ -0,0 +1,107 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+	<parent>
+		<groupId>org.eclipse.rdf4j</groupId>
+		<artifactId>rdf4j</artifactId>
+		<version>3.6.1-SNAPSHOT</version>
+	</parent>
+	<properties>
+		<spring.boot.version>2.4.3</spring.boot.version>
```

Comment:
We have a general Spring framework version set in the  root pom: `<spring.version>5.2.12.RELEASE</spring.version>`. Is Spring Boot on an independent release cycle?

Reply:
Spring-boot 2.4.3 depends on spring-framework 5.3.4. I can try to make an upgrade independent of this issue.

Reply:
I'll try to create the spring CQs today or tomorrow

Reply:
@barthanssens I'm not sure you aware but have a look at the section on ClearlyDefined in the project handbook (see https://www.eclipse.org/projects/handbook/#ip-clearlydefined): basically if a version of a library is known on clearlydefined.io and its license score is at least 75, we don't have to log a CQ for it.

Reply:
Interesting, thanks for the heads-up

But no such luck I guess, e.g. https://clearlydefined.io/?sort=releaseDate&sortDesc=true&name=spring-context 

If I understood the scoring correctly, spring context 5.3.4 has a "license" score of only 60  ("effective") OR 18
(Looks like there are multiple sources for the same artifact: the maven central and apache repo...)

Some other components may score slightly higher, some others worse....

Anyway, the CQs are approved, so either way we're good.

Reply:
> no such luck I guess

fiddlesticks